libsolidity/analysis/NameAndTypeResolver.cpp

```diff
@@ -425,6 +426,11 @@ vector<_T const*> NameAndTypeResolver::cThreeMerge(list<list<_T const*>>& _toMer
 	return result;
 }
 
+string NameAndTypeResolver::similarNameSuggestions(ASTString const& _name) const
+{
+    return quotedAlternativesList(m_currentScope->similarNames(_name));
```

Comment:
Since this has been split into utils I think `similarNameSuggestions` should only return a vector and the formatting of the contents should be in `ReferencesResolver`.

Reply:
Indentation.